### DISCUSSION TITLE
Import model missing texture

### DISCUSSION THREAD

**Discussion by biwuyi** [mozilla/hubs#6020] [Art Questions] [answered]
I downloaded it from 3D viewer Import .glb model into spoke，Model missing texture
![微信图片_20220121170103](https://user-images.githubusercontent.com/87455764/150497994-5c1ba987-a107-4685-9983-d8a9f4cb07a3.png)
![微信图片_20220121170109](https://user-images.githubusercontent.com/87455764/150497998-caf368d1-33f3-42e0-9be9-edff3e5683b2.png)
.

**Comment by Leticiazb** [comment]
Ola, boa noite, estou com o mesmo problema, consigo importar o arquivo dentro do spoke mas ele vem sem texturas.. voce ja teve algum avanço depois dessa postagem
ja importei do 3dsmaz, blender.. todos vem sem textura

**Comment by matthewbcool** [comment, marked as answer]
Not all materials/shaders are supported: https://docs.blender.org/manual/en/2.80/addons/io_scene_gltf2.html 

Take a look at the shader editor in the screenshots below for how I made adjustments. 
![5dc6b0f335665e3f4779f155c5d73b26](https://user-images.githubusercontent.com/4493657/164141627-09c42042-e1d8-4bd9-b0db-882b36d130ff.png)
![326bdb52679b828b12220fe4117dfc9a](https://user-images.githubusercontent.com/4493657/164141642-b261cf29-3984-426f-b631-55e6986858fd.png)
![cecb1ac04094e6d84208657b9ac58249](https://user-images.githubusercontent.com/4493657/164141644-f43b7721-1fa7-4068-a502-6b07e43f0781.png)
You could go further and tweak other aspects as well.
